kicad/AIOsense/AIOsense.kicad_pcb

```diff
@@ -12498,10 +12498,4 @@
       f6fe39d4-afed-4ca2-88e0-8e2ceaaf3e99
     )
   )
-  (group "" (id e453a6cb-b766-40db-b5bd-4a4b5cdf469d)
```

Comment:
Is there a reason for the ungrouping? Otherwise I would love to stay with my groups :)

Reply:
To be honest - I don't know what I did "ungroup" ? I even dond't know about a grouping function in KiCad? Cheers mate

Reply:
So we both learn something new about kicad :)

I can't change your PR, so you have to change it before i can approve.